### ISSUE TITLE
Hide module if format "resolves" to being empty

### ISSUE THREAD

**Issue by tardypad** [Alexays/Waybar#699]
Here is the config for my "capture" module
```
"pulseaudio#capture": {
  "format": "{format_source}",
  "format-source": "",
  "format-source-muted": ""
}
```
Its goal is to display the microphone icon only when it is not muted.
It works as intended. However when it is muted, the module is still present in the bar, showing nothing but still taking its space (as tested by setting a different background color for it)

My mpd module has the same issue
```
"mpd": {
  "format": "{stateIcon}",
  "format-stopped": "",
  "format-disconnected": "",
  "state-icons": {
    "paused": "",
    "playing": ""
  }
}
```
The module is still present when the server is stopped.

From what I see other modules I have seems correctly hidden when their format is empty.
My guess here is that the problem comes from the fact that the "format" key is not directly empty but "resolves" to it after some logic is applied. I believe the modules should be completely hidden in such cases as well.

**Comment by Alexays** [comment]
Will be fixed with #676

**Comment by WhyNotHugo** [comment]
I've the exact same use case. Have you found any workaround in the meantime?  
It seems that even if the source is muted, no additional css class is added to the module either, right?
